R/irods-rest.R

```diff
@@ -1,3 +1,15 @@
+#' Internal function to coordinate iRODS REST call
+#'
+#' @param endpoint REST API end-point.
+#' @param verb The HTTP verb to be used (e.g. PUT).
+#' @param args The arguments to be set for the end-point.
+#' @param verbose Verbosity of the HTTP request.
+#' @param object Data to send along with HTTP request.
+#' @param error Should status codes >400 be translated to errors.
```

Comment:
(In alignment with other boolean args around the package)
"Whether status codes larger than 400 should be translated to errors."

Reply:
changed as well